src/index.js

```diff
@@ -0,0 +1,79 @@
+import './index.css';
+import { deleteDataTodoRow } from './moduls/delete.js'; // eslint-disable-line
+import updateData from './moduls/update.js'; // eslint-disable-line
+import showData from './moduls/read.js'; // eslint-disable-line
+import completedTask from './moduls/completed.js'; // eslint-disable-line
+
+export const title = document.getElementById('title'), // eslint-disable-line
+  submit = document.getElementById('submit'),
+  updateDeleteItem = document.querySelector('.todo-tbody'),
+  clearDone = document.getElementById('clear-done');
+
+export let mood = 'creat'; // eslint-disable-line
+export let temp; // eslint-disable-line
```

Comment:
Please note that it would save you a lot of time if you place the comment : `/* eslint disable */` at the top of the page instead of putting `// eslint-disable-line` at the end of all these lines.